src/SFA-DAS-HMRC-API-Stub/Commands/GetFractionCalDateCommand.cs

```diff
@@ -0,0 +1,33 @@
+﻿using SFA.DAS.HMRC.API.Stub.Data.Repositories;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.HMRC.API.Stub.Commands
+{
+    public class GetFractionCalDateCommand : ICommand<GetFractionCalDateRequest, GetFractionCalDateResponse>
+    {
+        private readonly IFractionCalDateRepository _fractionCalDateRepository;
+
+        public GetFractionCalDateCommand(IFractionCalDateRepository fractionCalDateRepository)
+        {
+            _fractionCalDateRepository = fractionCalDateRepository ?? throw new ArgumentException("FractionCalDateRepository cannot be null");

```

Comment:
ArgumentException("FractionCalDateRepository cannot be null"); should be:
ArgumentException("fractionCalDateRepository cannot be null"); as the name should be the same as the parameter passed to the ctor